### ISSUE TITLE
Global nav visual style update (accessible color)

### ISSUE THREAD

**Issue by interactivellama** [salesforce/design-system-react#373]
The current Salesforce product color palette does not meet minimum accessibility standard, and if altered to do so, does not properly reflect the Salesforce brand.

Outcome:
After partnering with the brand/marketing team, and working through various options to make the solid background color work, we concluded that we needed a different design to solve for all our current and future requirements around branding/color systems/iconography, accessibility, future Salesforce product additions etc. 

The reason the change is a good thing:
- Splits accessibility requirements from branding needs
- Allows for true branding colors to be used by us (and our customers)
- Is aligned with the future of our brand vision (new iconography / work spaces etc)

Please refer to emails from Aran Rhee and others.
